CHANGELOG.md

```diff
@@ -3,6 +3,94 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.33](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.32...v4.0.0-alpha.33)
+
+_July 1, 2021_
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+- 💅 Allow to copy the selected rows to the clipboard with <kbd>CTRL</kbd> + <kbd>c</kbd> (#1929) @m4theushw
+- 🐛 `Select all` checkbox click should select only filtered rows (#1879) @ZeeshanTamboli
+- ⚡️ Add a new `singleSelect` column type (#1956) @DanailH
+
+  Using the column `type: 'singleSelect'` defaults to `Select` component when the cell is in `edit` mode. You can find the documentation [following this link](https://material-ui.com/components/data-grid/columns/#column-types).
+
+  ```
+    <DataGrid
+      columns={[
+        {
+          field: 'country',
+          type: 'singleSelect',
+          valueOptions: ['France', 'Netherlands', 'Brazil'],
+          editable: true,
+        }
+      ]}
+      rows={[
+        { id: 0, country: 'France' },
+        { id: 1, country: 'Netherlands' },
+        { id: 2, country: 'Brazil' },
+      ]}
+    />
+  ```
+
+### @material-ui/x-grid@v4.0.0-alpha.32 / @material-ui/data-grid@v4.0.0-alpha.32
+
+#### Breaking changes
+
+- [DataGrid] Rename `onColumnResizeCommitted` to `onColumnWidthChange` (#1967) @m4theushw
+
+  ```diff
+  -<DataGrid onColumnResizeCommitted={...} />
+  +<DataGrid onColumnWidthChange={...} />
+
+- [DataGrid] Make GRID_ROWS_CLEAR private (#1925) @oliviertassinari
+
+  The `GRID_ROWS_CLEAR` event was always triggered alongside `GRID_ROWS_SET`. You can listen to the latter event only.
```

Comment:
It's weird that we use different conventions in the same release to take about the events.

Reply:
True, let's keep only the camelCase version.